### PR TITLE
populate file: add exclude option

### PR DESCRIPTION
**Pull request by Mic92**

### CODE REVIEW

File: pkgs/populate/default.nix [hunk 1]
```diff
@@ -158,6 +158,9 @@ let
     ${rsync}/bin/rsync \
         ${optionalString (config.useChecksum or false) /* sh */ "--checksum"} \
         ${optionalString target.sudo /* sh */ "--rsync-path=\"sudo rsync\""} \
+        ${concatMapStringsSep " "
+          (pattern: /* sh */ "--exclude ${quote pattern}")
+          (config.exclude or [])} \
```
Comment (Lassulus) [inline pkgs/populate/default.nix]: is this or required? [] is the default anyway?

Comment (Mic92) [reply pkgs/populate/default.nix]: Not really. I just made it consistent with the usage `config.useChecksum` above. Should I remove both fallbacks?

Comment (Lassulus) [reply pkgs/populate/default.nix]: sounds like a good idea

**Comment by Mic92** [comment]
fixed!

**Pull request merged**